### DISCUSSION TITLE
[IntelliSense] Not working for PHP

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#11875] [Help]
Hello, thank you for your amazing work.

I'm tyring this extension for PHP file...I already add the below code in the `setting.json`:

`
"tailwindCSS.includeLanguages": {
   "php": "php"
}
`

The problem is that I don't know why most for the time the autocompletion / suggestion won't show.
I follow the official guild here for PHP `https://code.visualstudio.com/docs/languages/php`.

Am I missing anything or it doesn't work correctly with PHP? Thank you in advanced.

**Comment by bradlc** [comment]
Hey @karchung0930. I'm going to convert this to a discussion until we can identify an issue with the extension itself.

Please can you share the extension output? You can reveal the output panel using the "Tailwind CSS: Show Output" command.

If you can share your full VS Code settings in JSON format that would be helpful as well.

**Reply by ghost** [reply]
@bradlc Sure, please check this out ( https://streamable.com/rwn17k )

My setting.json:
```
{
    "window.commandCenter": false,
    "workbench.layoutControl.enabled": false,
    "workbench.colorTheme": "Dracula Pro",
    "workbench.startupEditor": "none",
    "files.autoSave": "onFocusChange",
    "editor.fontSize": 15,
    "editor.formatOnSave": true,
    "editor.minimap.enabled": false,
    "breadcrumbs.enabled": false,
    "editor.wordWrap": "on",
    "explorer.confirmDelete": false,
    "editor.fontFamily": "Fira Code Retina",
    "editor.fontLigatures": true,
    "git.ignoreMissingGitWarning": true,
    "explorer.confirmDragAndDrop": false,
    "tailwindCSS.includeLanguages": {
        "php": "php"
    }
}
```

**Reply by bradlc** [reply]
Super weird 🤔 Can you share a list of extensions you have installed? You can run `code --list-extensions --show-versions` if you have the VS Code CLI installed.

**Comment by ghost** [comment]
@bradlc I only have two extensions installed. One is Dracula Pro and one is PHP Intelephense, but I don't think they are the reason because if so, the HTML won't work either. Now only the PHP file not working, the HTML is working. You can try and test on your end. Thank you.

**Comment by bradlc** [comment]
Ah, I think it's because you don't have `editor.quickSuggestions` configured as [recommended](https://github.com/tailwindlabs/tailwindcss-intellisense#recommended-vs-code-settings):

```
"editor.quickSuggestions": {
  "strings": "on"
}
```

**Reply by kartikkumar01** [reply]
Thankyou so much. It started working now

**Comment by ghost** [comment]
@bradlc I just tested and you're right, thank you so much!
